<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>680</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>680</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu Light</family>
    <pointsize>12</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: #196D94;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="capturaProcesos">
       <property name="font">
        <font>
         <family>Ubuntu Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1">
           <widget class="QSpinBox" name="processNumberSP">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Black</family>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" rowspan="2">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="1">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="processAcountLB">
            <property name="minimumSize">
             <size>
              <width>96</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>96</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>No. Procesos:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="quantumSP">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Black</family>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="quantumLabel">
            <property name="minimumSize">
             <size>
              <width>96</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>94</width>
              <height>26</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Quantum:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QPushButton" name="processAcountPB">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Black</family>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(191, 191, 191);</string>
            </property>
            <property name="text">
             <string>ACEPTAR</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="programName">
          <property name="maximumSize">
           <size>
            <width>231</width>
            <height>18</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tw Cen MT</family>
            <pointsize>14</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Round-Robin</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mostrarProcesos">
       <property name="maximumSize">
        <size>
         <width>1270</width>
         <height>720</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="2" column="1">
         <widget class="QGroupBox" name="blockGB">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>160</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>160</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>17</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Bloqueados</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTableWidget" name="blockTB">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color: black;</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>TTB</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="processRuningGB">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>261</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>17</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Proceso en ejecución</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QTableWidget" name="processRuningTB">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color: black;
</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>36</number>
             </attribute>
             <attribute name="verticalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
             <row>
              <property name="text">
               <string>ID</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>OPE</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>TME</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>TT</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>TR</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>QUANTUM</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2">
         <widget class="QGroupBox" name="readyProcessGB">
          <property name="maximumSize">
           <size>
            <width>255</width>
            <height>430</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>17</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Listos</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QTableWidget" name="readyProcessesTB">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color: black;</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>32</number>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>70</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>TME</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>TT</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="newProcessesLB">
            <property name="maximumSize">
             <size>
              <width>169</width>
              <height>23</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>20</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>NUEVO</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="newProcessesLCD">
            <property name="maximumSize">
             <size>
              <width>64</width>
              <height>23</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="2" rowspan="2">
         <widget class="QGroupBox" name="finishedGB">
          <property name="maximumSize">
           <size>
            <width>430</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>17</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Terminados</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QTableWidget" name="finishedTB">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color: black;</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Operacion</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Resultado</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="globalCountLB">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>20</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>RELOJ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="globalCountLCD"/>
          </item>
         </layout>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="startRunPB">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>18</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: black;
background-color: white;</string>
          </property>
          <property name="text">
           <string>Comenzar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLabel" name="titleLB">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>15</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>ALGORITMO DE PLANIFICACION RR (ROUND-ROBIN)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mostrarTiempos">
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <widget class="QGroupBox" name="timesProcessesGB">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>20</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>TIEMPOS DE LOS PROCESOS</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QTableWidget" name="timesProcessesTB">
             <property name="font">
              <font>
               <family>Ubuntu</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: white;
color: black;
alternate-background-color: #606060;
selection-background-color:#282828;
</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <column>
              <property name="text">
               <string>ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Operacion</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Resultado</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T. Estimado</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T. Llegada</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T. Finalizacion</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T. Retorno</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T. Respuesta</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T. Espera</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T. Servicio</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Estado</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
